docs/setup/settings.asciidoc

```diff
@@ -52,67 +96,31 @@ The following example shows a valid regionmap configuration.
               - name: "INSEE"
                 description: "INSEE numeric identifier"
 
-`elasticsearch.username:` and `elasticsearch.password:`:: If your Elasticsearch is protected with basic authentication,
-these settings provide the username and password that the Kibana server uses to perform maintenance on the Kibana index at
-startup. Your Kibana users still need to authenticate with Elasticsearch, which is proxied through the Kibana server.
+`server.basePath:`:: Enables you to specify a path to mount Kibana at if you are running behind a proxy. This only affects
+ the URLs generated by Kibana, your proxy is expected to remove the basePath value before forwarding requests
+ to Kibana. This setting cannot end in a slash (`/`).
+`server.customResponseHeaders:`:: *Default: `{}`* Header names and values to send on all responses to the client from the Kibana server.
+`server.defaultRoute:`:: *Default: "/app/kibana"* This setting specifies the default route when opening Kibana. You can use this setting to modify the landing page when opening Kibana.
+`server.host:`:: *Default: "localhost"* This setting specifies the host of the back end server.
+dashboards. Kibana creates a new index if the index doesn’t already exist.
```

Comment:
Something happened here

Reply:
Good catch, that is supposed to be under `kibana.index`